jsonnet/overrideDashboard.libsonnet

```diff
@@ -0,0 +1,47 @@
+local overrideDashboard(dashboard, namespace, commonLabels, newDatasource) = 
+  dashboard {
+    metadata+: {
+      namespace: namespace,
+      labels: std.mergePatch(dashboard.metadata.labels, commonLabels),
```

Comment:
I'm curious why we use mergePatch instead of `:+`?

Reply:
Was trying something, removed :)